internal/containerztest/containerztest.go

```diff
@@ -0,0 +1,246 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package containerztest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/openconfig/containerz/client"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/ondatra"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	cpb "github.com/openconfig/gnoi/containerz"
+)
+
+// Client returns a new containerz client.
+func Client(t *testing.T, dut *ondatra.DUTDevice) *client.Client {
+	t.Helper()
+	gnoiClient := dut.RawAPIs().GNOI(t)
+	switch dut.Vendor() {
+	case ondatra.ARISTA:
```

Comment:
Could you add a switch case for Nokia? This should simply break as there is no config needed. Perhaps move the wait to the end of the Arista block.